trunk/examples/Backtranslation/regression/c/standard/test06_false-unreach-call_GenericAutomizerC_Generic.errorpath

```diff
@@ -9,21 +9,21 @@
 [L34]  EXPR        \read(a)
        VAL         [a={6:0}, p1={5:0}, p2={2:0}]
 [L34]  CALL, EXPR  f(a, &p1, &p2)
-       VAL         [\old(a)=4, p1={5:0}, p2={2:0}]
+       VAL         [\old(a)=4, \old(p1)={5:0}, \old(p2)={2:0}]
```

Comment:
Do you know why we still sometimes have old and "current" values, and sometimes both (e.g., see below)

Reply:
It looks like there was a bug in the old backtranslation s.t. we did not translate old-pointers properly. See also in line 14 of the same file `[\old(a)=4, p1={5:0}, p1={5:0}, p2={2:0}, p2={2:0}]`, where there are two entries for `p1` and `p2` each, I presume one of those entries should be old variables.

The variables that are present in the states depend highly on the settings that are used (Kojak/Automizer, with/without blockencoding, ...).

Reply:
Yes, but these tests should use single statement

Reply:
This PR should not change anything fundamental, it only fixes pointer-old-vars. The maybe unexpected values come probably from Matthias' Live CFG.